tox.ini

```diff
@@ -1,44 +1,67 @@
 [tox]
 minversion = 3.7.0
 envlist =
+    build-docker
+    doc
+    format-check
+    format-enforce
     lint
+    metadata-validation
     py{27,35,36,37}-ansible{25,26,27}-{functional,unit}
-    format-check
-    doc
 skipdist = True
 skip_missing_interpreters = True
 
 [testenv]
-usedevelop = True
+
 passenv = *
 setenv =
     PYTHONDONTWRITEBYTECODE=1
 deps =
-    -rrequirements.txt
-    -rtest-requirements.txt
     ansible24: ansible>=2.4,<2.5
     ansible25: ansible>=2.5,<2.6
     ansible26: ansible>=2.6,<2.7
     ansible27: ansible>=2.7,<2.8
     ansibledevel: git+https://github.com/ansible/ansible.git
+    build-docker: pbr
+    format: yapf>=0.25.0,<2
```

Comment:
It's not clear to me how `format-check`/`format-enforce` actually end up getting `yapf` installed. Maybe define deps with format-{check,enforce} to make it explicit? Not sure what else that might break, though. Also `format-enforce` is great.

```diff
diff --git a/tox.ini b/tox.ini
index e81580db..386ca54c 100644
--- a/tox.ini
+++ b/tox.ini
@@ -3,8 +3,7 @@ minversion = 3.7.0
 envlist =
     build-docker
     doc
-    format-check
-    format-enforce
+    format-{check,enforce}
     lint
     metadata-validation
     py{27,35,36,37}-ansible{25,26,27}-{functional,unit}
@@ -23,7 +22,7 @@ deps =
     ansible27: ansible>=2.7,<2.8
     ansibledevel: git+https://github.com/ansible/ansible.git
     build-docker: pbr
-    format: yapf>=0.25.0,<2
+    format-{check,enforce}: yapf>=0.25.0,<2
     {functional,unit}: -rtest-requirements.txt
     lint: -rlint-requirements.txt
     metadata-validation: collective.checkdocs
```

Reply:
nice, adopted